students/nDruP/lesson09/donor.py

```diff
@@ -0,0 +1,54 @@
+class Donor():
+    def __init__(self, title, gift):
+        self._name = title
+        self._history = []
```

Comment:
The leading underscore denotes a 'protected' attribute (only available for modification by class and subclass) which is not the case here, since these are being accessed and modified in `DonorDict`